packages/core/src/tweening/interpolationFunctions.ts

```diff
@@ -102,7 +123,7 @@ export function deepLerp(from: any, to: any, value: number): any {
       if (from instanceof Map && to instanceof Map) {
         const result = new Map();
         for (const key of new Set([...from.keys(), ...to.keys()])) {
-          const inter = deepLerp(from.get(key), to.get(key), value);
+          const inter = deepLerp(from.get(key), to.get(key), value, true);
```

Comment:
Shouldn't the value of `suppressWarnings` be passed here recursively?

Reply:
No. I'm intentionally suppressing warnings on `Map`s, where we intentionally use the nullish behavior.